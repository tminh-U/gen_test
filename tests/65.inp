12
16 1 20 20 5 2 9 10 10 9 5 18 
9
1 3
6 9
2 4
12 12
11 12
6 8
7 9
7 8
11 12
