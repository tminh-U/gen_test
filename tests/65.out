6
16
8
2
4
8
8
4
4
